pynio/block.py

```diff
@@ -13,7 +13,7 @@ def __init__(self, name, type, config=None, instance=None):
         self._name = name
         self._type = type
         self._template = None
-        self._config = config or {}
+        self._config = deepcopy(config) or {}
```

Comment:
This line was causing some unit tests to pass that shouldn't have before. Basically, the `config` variable was wrong in `mock.py